ISISComputingGroup/EPICS-ioc
# Ticket 5688: Newport controller

### Description of work

Created IOC for the Newport controller. Documentation at https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Newport. I added no tests as there is no emulator so any tests would basically only be testing against simulated axes, which we already test against in the galil tests.

### To test

* Run up in DEVSIM and confirm it creates simulated axes
* Ask me for connection details for the real hardware (will probably need to be done with someone in the office, I suspect a 30 minute ZOOM of "If I press this button does it move" is sufficient)
---

#### Code Review

    - [PV naming](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PV-Naming).
    - [Disable records](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Disable-records)
    - [Record simulation](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Record-Simulation)
    - [Finishing touches](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/IOC-Finishing-Touches)

### Functional Tests

- IOC responds correctly in:

### Final steps
